Analytics/resources/test_request_for_attribute.py

```diff
@@ -34,16 +35,25 @@ def test_attribute_params(test_client, attribute_param_value):
 	data = "attribute"
 	param = "attribute"
 
-	response_fcc_enpoint = test_client.get('/data?{}={}'.format(param,attribute_param_value))
-	response_dotmodus_enpoint = test_client.get('/data/{}?{}={}'.format(data,param,attribute_param_value))
+	response_fcc_enpoint = test_client.get(
+		'/data?{}={}'.format(
+			param,
+			attribute_param_value))
```

Comment:
Please put the close brackets on a new line so it is easier to see where the function ends